shareprogress/services.py

```diff
@@ -0,0 +1,35 @@
+# Managing Buttons
+def create_button(page_url, button_template, page_title=None, *args):
+    # page_url: The URL of the page to be shared.
```

Comment:
Done! (Thought I'm wondering if I need to delete the lines between each argument explanation)